app/battle/knights_fight.py

```diff
@@ -0,0 +1,20 @@
+from app.knights.prepare_to_battle import KnightsPrepareToBattle
+
+
+class BattleRound:
+
+    @staticmethod
+    def knights_battle_result(first_knight: KnightsPrepareToBattle,
+                              second_knight: KnightsPrepareToBattle
+                              ) -> dict:
```

Comment:
Change identation. If it doesn't fit into 1 line, you can leave the parantheses and arguments in separate lines:

```
def knights_battle_result(
    first_knight: KnightsPrepareToBattle,
    second_knight: KnightsPrepareToBattle
) -> dict: